ouroboros-consensus/src/Ouroboros/Consensus/Block/Forging.hs

```diff
@@ -218,20 +211,8 @@ checkShouldForge BlockForging{..}
 -------------------------------------------------------------------------------}
 
 -- | The result of updating something, e.g., the forge state.
-data UpdateInfo updated unchanged failed =
-    Updated      updated
-  | Unchanged    unchanged
```

Comment:
Just an FYI: this *might* affect the tracing of the KES info in the node. However, I think that because the HFC has no "unchanged" case (see https://github.com/input-output-hk/ouroboros-network/pull/2820), this change is probably invisible.

I guess this is more of a warning for when people start complaining (again?) about the node tracing the KES metrics every leadership check (= every slot) instead of when they changed. It used to be the latter (traced when changed), but because they want them traced at startup so that the RTView can pick them up (communicates via tracing), it was changed to be traced at every leadership check. Otherwise the RTView would have to wait until they actually change (after slotsPerKESPeriod = 129600 slots = 36 hours).

Reply:
Thanks for the note. (We have agreed to deal with this later, whenever it comes up, if it does.)